Improve the explanation used for getInitialProps (#476)
zeit/next-site: pages/learn/basics/fetching-data-for-pages/index.mdx

In practice, we usually need to fetch data from a remote data source. Next.js comes with a standard API to fetch data for pages. We do it using an async function called `getInitialProps`.

> `getInitialProps` can only be added to the default component exported by a page, adding it to any other component won't work.

> With `getInitialProps`, we can fetch data for a given page via a remote data source and pass it as props to our page. Our `getInitialProps` should work on both server and client; So, Next.js can use it on both server and client.

Reviewer: ```suggestion
With `getInitialProps`, we can fetch data for a given page via a remote data source and pass it as props to our page. `getInitialProps` needs to work on both server and client; since it is called in both environments.
```
